OrbitGl/PickingManager.cpp

```diff
@@ -7,28 +7,37 @@
 #include "OpenGl.h"
 #include "OrbitBase/Logging.h"
 
-PickingId PickingManager::CreatePickableId(std::weak_ptr<Pickable> a_Pickable,
-                                           BatcherId batcher_id) {
+PickingId PickingManager::CreateOrGetPickableId(
+    std::weak_ptr<Pickable> pickable, BatcherId batcher_id) {
```

Comment:
this weak_ptr is never checked - it may result in nullptr dereference in line 14.

Reply:
Nit: GetOrCreate is probably better. It creates only if was not able to get.

Reply:
Done